### PR TITLE
updated main page

### PR DESCRIPTION
**Pull request by Smullz622**
removed gpa and added yearly summary

### CODE REVIEW

File: app/views/main/index.html.erb [hunk 1]
```diff
@@ -61,12 +61,11 @@
       <li>From the integration screen, upload this file where it says "Courses taught data file"</li>
       <li>Enter the passcode and click the button corresponding to the target environment you wish to integrate with</li>
     </ol>
-    <li><h5>GPA Data Integration</h5></li>
+    <li><h5>Yearly Data Integration</h5></li>
     <ol>
-      <li>Download the latest GPA data.  This will be stored in box and obtained from the College of Liberal Arts.</li>
-      <li>Make sure the headers and fields are formatted in the following way:<img src="gpa-headers.png" alt="headers example" width="1100" height="40"></li>
```
Comment (ajkiessl) [inline app/views/main/index.html.erb:67]: We should remove the `public/gpa-headers.png` file too.

**Review by ajkiessl — CHANGES_REQUESTED** [review]
I have one request below.  Also, is the plan to retire the GPA integration entirely? It might be a good idea to remove all of its code; if it will no longer be used.

**Comment by Smullz622** [comment]
I checked with Nicole, and yes it is retired. I removed all the code related to it with the exception of the schema. I just wanted to double check it was ok to do so before removing the table & related data.

**Comment by ajkiessl** [comment]
There probably isn't any data being stored for GPA out in production, so it's okay to remove.  I haven't touched this stuff in a while though, so it might be a good idea to double-check on the server.

**Review by ajkiessl — APPROVED** [review]

**Pull request merged**